docs/user/guide-intermediate.md

```diff
@@ -0,0 +1,323 @@
+# Guide to intermediate level Cursorless
+
+This document will try to:
+
+- Highlight the most useful actions and modifiers in Cursorless so you know which ones to focus on learning first.
+- Bring together a few important concepts that underlie a lot of Cursorless behavior.
+- Walk through some example editing operations that are common but perhaps tricky to a beginner.
+
+This document assumes that you understand the very basics of cursorless, like what commands `"take air"` and `"chuck line"` do.
+If you don't have the basics down, please read some of [the main documentation page](README.md) and/or watch [the Cursorless tutorial videos](https://www.youtube.com/watch?v=5mAzHGM2M0k&list=PLXv2sppxeoQZz49evjy4T0QJRIgc_JPqs).
+
+It is planned for this guide to be expanded to both basic and advanced levels.
+
+## Important actions
+
+Here are some of the most useful actions.
+Parts in square braces (`[]`) are optional.
+Slashes (`/`) show alternatives.
+Targets are represented by `T` with a possible digit.
+
+- Selection Manipulation
+  - `"take T"`: set selection.
+  - `"pre/post T"`: set selection before/after T.
+- Clipboard
+  - `"paste to/before/after T"`.
+  - `"carve/copy T"`: cut/copy T.
+- Core Changers
+  - `"bring T"`: insert a copy of T at the cursor/selection.
+  - `"bring T1 to T2"`: replace T2 with T.
+  - `"bring T1 before/after T2"`: insert a copy of T1 before/after T2, including appropriate delimiters.
+  - `"move T1 [to/before/after T2]"`: like `"bring"`, but moves instead of copies.
+  - `"chuck T"`: delete T and appropriate delimiter.
+  - `"change T"`: delete T and set cursor(s) to where T was; delimiters are unchanged.
+  - `"drink/pour T"`: edit new line before/after T.
+- IDE-related
+  - `"follow T"`: open URL or go to definition of T.
+  - `"quick fix T"`: show quick fix for T.
+  - `"reference T"`: show references to T.
+
+## Important modifiers
+
+Here are some very useful modifiers:
+
+- Syntactic scopes ([reference](README.md#syntactic-scopes)) are great for expanding the target to what you want to operate on, like the containing function.
```

Comment:
The link is to the syntactic scopes section, but you mention non-syntactic scope modifiers, like line and block.  Perhaps I am structuring things too much according to the reference page, but I fear these changes will confuse people when they try to explore the reference page.

I can still move the link to the "Syntactic scopes" and delete the "(reference)".

Reply:
The reference page should change. We used to draw a distinction between "syntactic" scopes and scopes like "line", but that is now ancient history. Unfortunately the docs haven't quite caught up 😅

If you wanted to draw a distinction, it would be between language-specific vs globally available, but I'm not sure that's worth highlighting at this point

Reply:
In the meantime, how about I merge the lists like you and have a preface more like `Scope modifiers (like [syntactic scopes](README.md#syntactic-scopes)) are great for expanding the target to operate on something bigger, like the containing line, function or block`?  I don't want to have a link that seems like it is for more than syntactic scopes.